packages/protocol/contracts/dapis/rrpDapiServer.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "../AirnodeRrp.sol";
+import "./CustomReducer.sol";
+
+contract RrpDapiServer is CustomReducer {
+    struct DapiParameters {
+        uint256 minResponsesToReduce;
+        int256 toleranceInPercentages;
+        uint256 requesterIndex;
+        bytes32[] templateIds;
+        address[] designatedWallets;
+        address reduceAddress;
+        bytes4 reduceFunctionId;
+        }
+    
+    AirnodeRrp public airnodeRrp;
+    mapping(uint256 => DapiParameters) private dapiIdToParameters;
+    uint256 public nextDapiID = 1;
```

Comment:
I tend to refer to `bytes32` identifiers as "ID" and number-type identifiers as "index"